src/styles/variables.scss

```diff
@@ -0,0 +1,5 @@
+$blue-accent: #00acdc;
+$white: #fff;
+$secondary: #616070;
+$main-accent: #060b35;
+$border: #f3f3f3;
```

Comment:
Using variables for colors is a good idea 👍 

However, I would suggest picking more meaningful names for colors. Aslo, avoid using color names here (imagine if designers will decide to change `blue-accent` to orange or green), but `moyo-blue-accent` can be valid, as it refers to some brand book.

suggestions:
white - background, card-background, ...
blue-accent - accent, moyo-blue, ...
border - card-border-color, primary/secondary-border-color (using "color" here, because it may be mistaken for the entire border setup )

and so on